README.md

```diff
@@ -73,9 +73,17 @@ and then you need to prepare this as a list of strings (surround it with double
 sed -i -e 's/\(.*\)/    "\1",/' qr_ascii.txt
 ```
 
+But there is no need to create files or execute three command lines . 
+
+#### Simple one-liner with output to stdout, replace SSID and PASSWD accordingly
```

Comment:
I'd use just `Simple one-liner with output to stdout`

Reply:
Comments to yourt comments:
1.) "No need for this line I think"
agreed

2.) "Shouldn't this use single backtick too?"

I used triple backquotes because of the automagically appended "copy" button to the right of the line.

3.) "Maybe $YOUR_SSID and $YOUR_PASSWORD is better?"

I used "read p1 p2" to not distract the user and clarify, where exactly to put the ssid and the password in the command string. Alas, I agree, maybe the line should begin with "echo YOUR_SSID_HERE YOUR_PASSWORD_HERE | ..."  without (!) the  prepended "$"of your suggestion and the two variables appended by "_HERE"

4.) "Simple one-liner with output to stdout"

agreed

Thanks for your quick response to the pull request and of course thanks for your QR-code generator.

BTW: I like your "create_new_password_and_regenerate_qr_code.sh". I didn't read it until now and so your README.md makes a lot more sense regarding this shell script. 

Cheers,
Hampf

Reply:
Yeah I forgot about the script myself...haven't used this one in a while :)